docs/Tutorials/Send-private-transaction.md

```diff
@@ -15,10 +16,11 @@ and send a private transaction.
 
 ## 1. Create private contract
 
-In the `Node-0` directory, copy and paste the following to a file called `private-config.js`. Replace
-the placeholder for `privateFor` with the `tessera1.pub` key.
+In the `Node-0` directory, copy and paste the following to a file called `private-contract.js`. On row
```

Comment:
For consistency, perhaps the indication should be `On the highlighted lines` instead of row and a number. But this is not a blocker for merge.